src/python/pants/init/options_initializer.py

```diff
@@ -48,15 +48,17 @@ def _initialize_build_configuration(
     """
 
     bootstrap_options = options_bootstrapper.get_bootstrap_options().for_global_scope()
-    backends_requirements = _collect_backends_requirements(bootstrap_options.backend_packages)
-    working_set = plugin_resolver.resolve(options_bootstrapper, env, backends_requirements)
 
     # Add any extra paths to python path (e.g., for loading extra source backends).
     for path in bootstrap_options.pythonpath:
         if path not in sys.path:
             sys.path.append(path)
             pkg_resources.fixup_namespace_packages(path)
```

Comment:
This was required to be moved above `_collect_backends_requirements` so that  the below `importlib.util.find_spec(backend_package)` will work, as the `sys.path` didn't include the necessary `[GLOBAL].pythonpath` values yet.

I'm guessing in the original PR, the author's repo just so happened that you could load the backend packages from repo root in vanilla Python.

Reply:
sneaky